ego4d/internal/human_pose/triangulator.py

```diff
@@ -103,6 +105,7 @@ def run(self, debug=False):
                         n_iters=self.n_iters,
                         reprojection_error_epsilon=self.reprojection_error_epsilon,
                         direct_optimization=True,
+                        sample_all_combinations=True
```

Comment:
nit: For backward (body) compatibility this should be an option with False as default value, or passed as an option during Triangulator initialization). Curious that if this option improves the results, or if it increases the time for this step :)

Reply:
This will help decrease the running time significantly - instead of randomly selecting two views as usually does in RANSAC (default), the new change would iterate through all possible combinations of camera views, which reduces the number of iterations needed from a large constant value (e.g. `n_iter=200`) to a usually small value (e.g. C(5,2)=10 assuming 4 exo camera views + 1 aria rgb view). Also, it guarantees to find the best two camera views as every possible ways of camera view combinations have been tested.

Reply:
That's great! Will try to land this code now